sorted_words.rb

```diff
@@ -0,0 +1,9 @@
+# type any words you want!
+puts "type any word"
+# user types in any word then sort 
+user_input = gets.chomp.split.sort
```

Comment:
The assignment reads:

> The user should be able to add as many words as they like, until they just hit enter to return a blank word.

This means the `gets.chomp` should be inside a loop that breaks when the input is blank.